eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -1,20 +1,31 @@
 {
+  "displayNames": {
+    "--disablecov": ""
+  },
   "matrix": {
     "Agent": {
-      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
```

Comment:
Based on our discussion yesterday I assume you still plan to add back the other OS's here to the base matrix as well, correct?